gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/domain_service/VerifyApiHostsDomainService.java

```diff
@@ -52,9 +53,12 @@ public class VerifyApiHostsDomainService {
         "^([a-z0-9]|[a-z0-9][a-z0-9\\-_]{0,61}[a-z0-9])(\\.([a-z0-9]|[a-z0-9][a-z0-9\\-_]{0,61}[a-z0-9]))*$"
     );
 
+    // Kafka host must have a first segment that is less than 50 characters
```

Comment:
Depending on the configuration of the gw it can be a little more or less. I don't know if it's a problem. To be seen with the use of our customers.

So 50 is a bit of a compromise value
#Just to know